server/subscribe.go

```diff
@@ -209,7 +209,9 @@ func (p *Plugin) getChannelsSubscribed(wh *webhook, instanceID types.ID) ([]Chan
 	subIds := subs.Channel.ByID
 	for _, sub := range subIds {
 		if p.matchesSubsciptionFilters(wh, sub.Filters) {
-			channelSubscriptions = append(channelSubscriptions, sub)
+			if !HasSubscription(sub, channelSubscriptions) {
```

Comment:
Why not use a map for avoid duplicates instead of walking the list for every append?

Reply:
Hey @wiggin77, I have implemented the suggestion. Please have a look.